packages/abi-coder/src/index.ts

```diff
@@ -2,7 +2,7 @@ export { Coder, InputValue, DecodedValue } from './encoding/coders/AbstractCoder
 export type { FunctionFragment } from './FunctionFragment';
 export * from './encoding/coders';
 export { Interface } from './Interface';
-export { JsonAbi, JsonAbiArgument } from './types/JsonAbi';
```

Comment:
This `JsonAbiArgument` was used only in the [changed](https://github.com/FuelLabs/fuels-ts/pull/2826/files#diff-e3d0c6c1264fe3529431371fe4d50cd6ef11a4b1baabbc774600053db5c764ccR59) documentation so I stopped exporting it.